packages/gatsby-cli/src/reporter/loggers/ink/context.js

```diff
@@ -1,6 +1,8 @@
 import React, { useState, useEffect, createContext } from "react"
 
-import { getStore, onLogAction } from "../../redux/index"
+import ReporterStore from "../../redux/index"
+
+const { getStore, onLogAction } = ReporterStore
```

Comment:
Any reason for doing it in 2 steps? 🤔

Reply:
No particular reason, @blainekasten fixed it by removing the object structure in the commits below, I highly prefer his solution.